science/mutationpp/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            mutationpp Mutationpp 1.0.0 v
+name                    mutationpp
+revision                0
+categories              science
+license                 LGPL-3+
+maintainers             {@rubendibattista} openmaintainer
+platforms               darwin
+description             The MUlticomponent Thermodynamic And Transport library for IONized gases in C++
+long_description        Mutation++ is an open-source library originally \
+                        developed at the von Karman Institute for Fluid Dynamics, designed to couple \
+                        with conventional computational fluid dynamics codes to provide thermodynamic, \
+                        transport, chemistry, and energy transfer properties associated with subsonic \
+                        to hypersonic flows.
+
+# This patch is needed in order to avoid that cmake uses `FindEigen3.cmake`
+# instead of the Eigen3Config.cmake that provides the right target
+patchfiles              eigen3-config.diff
+
+checksums               rmd160  477c2da3eb49ab41fc64b3995f059511ed82e421 \
+                        sha256  2efeebf39f4f69cd98fd55c296b07726071ec0bcf7663a87571af8912e591a95 \
+                        size    4020173
+
+depends_lib-append      port:eigen3
```

Comment:
Can `eigen3-devel` also work here? If so, use this instead:

`depends_lib-append      path:include/eigen3/Eigen/Eigen:eigen3`

Reply:
I need the `Eigen3Config.cmake` file. Does `eigen3-devel` installs it?

Reply:
It seems to work on my machine, I guess we'll need to wait the pipelines to be sure.